lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -622,6 +622,10 @@ MStatus MtohRenderOverride::Render(
         _mayaHydraSceneIndex->PreFrame(drawContext);
     }
 
+    if (_displayStyleSceneIndex) {
```

Comment:
I think you should wait for my PR to be merged (it's ready to be merged) as there are functions to handle different cases for display styles and you could put the creation of this scene index in there.